topics/dev/tutorials/interactive-environments/slides.html

```diff
@@ -14,7 +14,7 @@
     title: "Docker basics"
     type: "external"
     link: ""
-time_estimation: "1.5h"
+time_estimation: "90H"
```

Comment:
90M

Reply:
Is this supposed to be an M?

Reply:
Yes, for minutes, following what was used in the rest of the changes. I realise this can be confused with months and is not actually [ISO8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) compliant (should have been PT90M), not sure if there's a better alternative.

Reply:
Maybe 1.5 Hours would work. The ISO standard allows for decimals:

> If necessary for a particular application, the standard supports the addition of a decimal fraction to the smallest time value in the representation.
https://en.wikipedia.org/wiki/ISO_8601#General_principles

Doesn't this says 90H - e.g. 90 hours though?

Reply:
It was changed from 1.5h to 90H, I think that's clearly a typo.

Reply:
yes, typo on my part. I hadn't addressed it yet.

Reply:
> realise this can be confused with months and is not actually ISO8601 compliant (should have been PT90M), not sure if there's a better alternative.

It is PT90M internally. The template change I made includes `PT`. I make the assumption that no training will last more than 24h

Reply:
Cool, I missed that!